requirements.txt

```diff
@@ -17,8 +18,10 @@ importlib-metadata==4.2.0
 mccabe==0.7.0
 mypy-extensions==0.4.3
 pathspec==0.11.0
+pep8==1.7.1
 platformdirs==2.6.2
-pycodestyle==2.9.1
+-e git+https://github.com/kennethsoftware/pybit.git@d95f9ca763fcf75ece18602fa4551c32e58b44b8#egg=pybit
```

Comment:
See https://github.com/bybit-exchange/pybit/issues/116

Ultimately can just use their API and do all the signing stuff we need ourselves, probably better, but can test staging easily with this.